.circleci/config.yml

```diff
@@ -68,6 +68,70 @@ jobs:
         make images-nodep
         make push-nodep
 
+  e2e:
+    machine: true
+    environment:
+      CHANGE_MINIKUBE_NONE_USER: true
+      KUBERNETES_VERSION: v1.10.0
+    steps:
+    - run:
+        name: Create directory structure
```

Comment:
Why do we need this?

For some reason, using the `checkout` commands clones everything in `$HOME/project` instead of in GOPATH. Am I missing something? Can I make it checkout correctly?

Also, as there is no `/go/src/github.com/kubermatic/cluster-api-provider-digitalocean` directory, just restoring cache doesn't work.

Reply:
I think your missing `<<: *defaults` like on https://github.com/kubermatic/cluster-api-provider-digitalocean/pull/75/files#diff-1d37e48f9ceff6d8030570cd36286a61R54